src/options.js

```diff
@@ -1,27 +1,32 @@
-module.exports = {
+/**
+ * @type {import("..").IOptions}
+ */
+const options = {
   path: "",
   globalClasses: "",
   defaultClasses: "",
   svgSpriteShortcode: "svgsprite",
   svgShortcode: "svg",
-  spriteConfig:
-  {
+  spriteWrap: null, // defaults to { height: '0', width: '0' }
```

Comment:
add the option here, it defaults to `null` respecting the previous versions approach of wrapping the `<svg>`. This way users can optionally decide whether or not they want to disable it.